test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -483,6 +513,20 @@
         (is (true? (fs/exists? signed-cert-path)))
         (is (= 204 (:status response))))))
```

Comment:
Yeah, duh, my bad.  I'd done this work against the `master` branch initially - which is why I had the new-style URLs in the tests - before remembering I probably should have done this against the stable branch instead.  But when I reapplied the changes to stable, I didn't change the URLs back to look like what they would for stable.  In any event, I'll replace the 'v1' with 'production' because I agree that looks better / more consistent with the other tests.